zmon_cli/client.py

```diff
@@ -63,6 +63,8 @@ def wrapper(*args, **kwargs):
 
 
 def compare_entities(e1, e2):
+    e1.pop('last_modified', None)
```

Comment:
But it makes sense not to change the input, at least this is not what the function supposed to do. Will modify it.